cypress/support/pages/settingPageObjects.js

```diff
@@ -0,0 +1,69 @@
+import SignInPageObject from "./signIn.pageObject";
+
+class SettinsPageObject extends SignInPageObject {
+  url = '/settings';
+
+  loginUser() {
+    cy.login();
+  }
+
+  updateUsername(username) {
+    cy.getByDataCy('username-settings')
+      .clear()
+      .type(username);
+  }
+
+  updateBio(bio) {
+    cy.getByDataCy('bio-settings')
+      .type(bio);
+  }
+
+  updateEmail(email) {
+    cy.getByDataCy('email-settings')
+      .clear()
+      .type(email);
+  }
+
+  updatePassword(password) {
+    cy.getByDataCy('password-settings')
+      .type(password);
+  }
+
+  clickSubmitButton() {
+    cy.getByDataCy('submit-settings')
+      .click();
+  }
+
+  assertBio(bio) {
+    cy.getByDataCy('bio-settings')
+      .should('contain', bio);
+  }
+
+  logOut() {
+    cy.getByDataCy('/settings')
```

Comment:
It looks like you are using getByDataCy with a URL path. This seems incorrect because the getByDataCy function is supposed to be used with a data-cy attribute, not a URL path.